cfgov/unprocessed/js/modules/util/dom-manipulators.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var domTraverse = require( './dom-traverse' );
+
+/**
+ * Shortcut for creating new dom elements
+ * @param   {string} tag     The html elem to create
+ * @param   {object} options The options for building the elem
+ * @returns {HTMLNode}       The created elem
+ */
+function create( tag, options ) {
+  var elem = document.createElement( tag );
```

Comment:
@sebworks Oh I see. Could I also just call `var queryOne = require( './dom-traverse' ).queryOne;` to reduce the require?